src/solve.jl

```diff
@@ -534,6 +532,15 @@ function DiffEqBase.__init(
     integrator
 end
 
+function DiffEqBase.__init(prob::ODEProblem, ::Nothing, args...; kwargs...)
+    alg = DiffEqBase.prepare_alg(DefaultODEAlgorithm(), prob.u0, prob.p, prob)
```

Comment:
you don't need to (and shouldn't) prepare the alg here.

Reply:
ah, I was basing off of https://github.com/SciML/DifferentialEquations.jl/blob/ad7b74d096f2883c5686c48a11f4edd94eef644e/src/default_solve.jl#L8. I'll make sure the tests still pass with it deleted.